.github/actions/rust-cargo-run/action.yml

```diff
@@ -0,0 +1,51 @@
+name: rust cargo run
+description: Rust Cargo Run
+
+inputs:
+  command:
+    description: The cargo command to execute
+    required: true
+  args:
+    description: The arguments to provide for the cargo command
+    required: false
+  components:
+    description: The components to include when installing Rust
+    required: false
+  github_token:
+    description: Github Token
+    required: true
+  save_cache:
+    description: Whether to save the SSCACHE
+    required: false
+  cache_name:
+    description: The name of the cache to save/restore
+    required: true
+    default: test
+
+runs:
+  using: composite
+  steps:
+  - name: Installing Rust
+    uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af # v1.0.7
+    with:
+      profile: minimal
+      target: wasm32-unknown-unknown
```

Comment:
dunno about that target, but I guess it works

Reply:
Er, yeah, don't need that.